### ISSUE TITLE
[Win7] Crash on exit (every time) with v0.9.6.0

### ISSUE THREAD

**Issue by Gitoffthelawn** [Fushko/gammy#65]
Thank you for Gammy.  It is unique, and overall, excellent.

With the latest release, v0.9.6.0, Gammy crashes every time it is terminated via its system tray item's context menu.

Details:
```
Problem signature:
  Problem Event Name:		APPCRASH
  Application Name:		gammy.exe
  Application Version:		0.9.6.0
  Application Timestamp:	5fef727f
  Fault Module Name:		gammy.exe
  Fault Module Version:		0.9.6.0
  Fault Module Timestamp:	5fef727f
  Exception Code:		c0000005
  Exception Offset:		0000000000002dde
  OS Version:			6.1.7601.2.1.0.768.3 (Win7)
```

Reproducibility: 100%

**Comment by Fushko** [comment]
Hello, please follow these steps. In the app folder:
1. Delete gammylog.txt
2. Open gammyconf.txt and set "log_level" to 6
3. Reproduce the crash 
4. Upload gammylog.txt here

**Comment by Gitoffthelawn** [comment]
May I send you the log via email?

**Comment by Fushko** [comment]
> May I send you the log via email?

Yes. But why not here? Just drag and drop.

**Comment by Gitoffthelawn** [comment]
I verified the crashing issue still exists in v0.9.6.1.

The information you requested is here: https://pastebin.com/Qy0eRBHB

The password is your first name.  Let me know when you've received it.

**Comment by Fushko** [comment]
> I verified the crashing issue still exists in v0.9.6.1.
> 
> The information you requested is here: https://pastebin.com/Qy0eRBHB
> 
> The password is your first name. Let me know when you've received it.

Thanks. The log doesn't show anything useful. I'll have to check myself.

**Comment by Gitoffthelawn** [comment]
You're welcome.  I didn't see anything useful in the log either, but I was hoping perhaps you would. :)

**Comment by Fushko** [comment]
Fixed on develop

**Comment by Fushko** [comment]
Fixed on 0.9.63

**Comment by Gitoffthelawn** [comment]
Confirmed.  No crash in 0.9.63.  Thank you!
